innopoints/schemas/activity.py

```diff
@@ -93,21 +102,13 @@ def get_existing_application(self, activity):
 
     working_hours = ma.Int(allow_none=True, validate=validate.Range(min=1))
     reward_rate = ma.Int(validate=validate.Range(min=1))
-    people_required = ma.Int(allow_none=True, validate=validate.Range(min=0))
+    people_required = ma.Int(allow_none=True, validate=validate.Range(min=1))
```

Comment:
So you allow them to create projects/activities with duplicate because "they know what they are doing", but they cannot create an activity that doesn't require volunteers?
I can already think of a use-case: documenting the entire project, but one activity is already done and doesn't need more work, but they want to add for the sake of completion and showing off what they did all in all

Reply:
Pretty pathetic use-case if you ask me, but okay, I see no obstacles in allowing it)